changelog/unreleased/4351

```diff
@@ -0,0 +1,7 @@
+Bugfix: Resolve incorrect truncation of long display names in Manage Accounts view
+
+Resolved the bug where long display names were truncated incorrectly in the Manage Accounts view.
+Now, display names are properly truncated in the middle with ellipsis (...) to maintain readability.
+
+Affected Issue:
+- Fixes #4351
```

Comment:
To keep the correct format of the Calens file, we must place at the end in 2 different lines the link of the issue and the link of the PR. In this case, this would be:
```suggestion
https://github.com/owncloud/android/issues/4351
https://github.com/owncloud/android/pull/4380
```